packages/imtr-client/src/models/checker.ts

```diff
@@ -312,4 +313,69 @@ export default class Checker {
     }
     return question || null;
   }
+
+  /**
+   * Return the outcomes
+   *
+   * @return {Outcome[]} An Array of Outcome objects
+   *
+   */
+  getOutcomesToDisplay(): Outcome[] {
+    return this.permits
+      .filter((permit: Permit) => !!permit.getOutputByDecisionId("dummy"))
+      .map((permit: Permit) => {
+        const conclusion = permit.getDecisionById("dummy") as Decision;
+        const conclusionMatchingRules = conclusion.getMatchingRules() as Rule[];
+        const contactOutcome = conclusionMatchingRules.find(
+          (rule) => rule.outputValue === imtrOutcomes.NEED_CONTACT
+        ) as Rule;
+        const outcome = (contactOutcome?.outputValue ||
+          conclusionMatchingRules[0].outputValue) as string;
+
+        return {
+          outcome,
+          title:
+            outcome === imtrOutcomes.NEED_CONTACT
+              ? "Neem contact op met de gemeente"
+              : `${permit.name.replace("Conclusie", "")}: ${removeQuotes(
+                  outcome
+                )}`,
+          description:
+            outcome === imtrOutcomes.NEED_CONTACT
+              ? contactOutcome.description
+              : conclusionMatchingRules[0].description,
+        };
+      });
+  }
+
+  /**
+   * Return the outcome type
+   *
+   * @returns {string} The type of the outcome as defined in `imtrOutcomes`
```

Comment:
Please remove the return type from doctypes. It doesn't add value and can be confusing when its not in sync with typescript.